content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/index.mdx

```diff
@@ -5,7 +5,7 @@ metaDescription: 'Learn how to create a new Node.js or TypeScript project from s
 duration: '15 min'
 toc: false
 langSwitcher: ['typescript', 'node']
-dbSwitcher: ['postgres', 'mysql', 'sqlserver']
+dbSwitcher: ['postgres', 'mysql', 'sqlserver', 'planetscale']
```

Comment:
This page is missing the "Next" button at the bottom for `planetscale`

Reply:
Oops, thanks. Done.